apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/avstemming/AvstemmingDao.kt

```diff
@@ -6,19 +6,22 @@ import kotliquery.queryOf
 import kotliquery.sessionOf
 import kotliquery.using
 import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Saktype
 import java.sql.Timestamp
 import javax.sql.DataSource
 
 class AvstemmingDao(private val dataSource: DataSource) {
 
+    // fun opprettKonsistensavstemming(konsistensavstemming: Konsistensavstemming): Int =
```

Comment:
Denne skal inn når alt er klart for konsistensavstemming?

Reply:
ja, er en miss, jeg fikser